git-find-lfs-extensions.py

```diff
@@ -62,7 +62,7 @@ def add_file(ext, type, size_mb):
     MAX_TYPE_LEN = max(MAX_TYPE_LEN, len(type))
     if ext not in result:
         result[ext] = {
-            'ext' : ext,
+            'ext' : ext.lower(),
```

Comment:
should we do `ext = ext.lower()` at the beginning of the function? otherwise we mess up the counting a few lines below, no?